From ad7ca22b6a5a9669ed9edd19a5b467ed6b9ead3c Mon Sep 17 00:00:00 2001
From: FT Liang <gkk886@gmail.com>
Date: Mon, 13 Oct 2014 11:12:17 +0800
Subject: [PATCH 5/6] ARM: dts: AM33XX: Fix system power off control in
 am335x-bone

Fix system power off control of rtc node on ported to 3.14 kernel,
which should be placed under ocp node to match am33xx.dtsi.
---
 arch/arm/boot/dts/am335x-bone-common.dtsi | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/am335x-bone-common.dtsi b/arch/arm/boot/dts/am335x-bone-common.dtsi
index 026e7f8..85caf75 100644
--- a/arch/arm/boot/dts/am335x-bone-common.dtsi
+++ b/arch/arm/boot/dts/am335x-bone-common.dtsi
@@ -195,10 +195,12 @@
 		ldo4_reg: regulator@6 {
 			regulator-always-on;
 		};
+	};
+};
 
-		rtc@44e3e000 {
-			ti,system-power-controller;
-		};
+&ocp {
+	rtc@44e3e000 {
+		ti,system-power-controller;
 	};
 };
 
-- 
2.1.1

